stdlib/private/SwiftPrivateLibcExtras/SwiftPrivateLibcExtras.swift

```diff
@@ -37,36 +37,36 @@ public var _stdlib_FD_SETSIZE: CInt {
   return 1024
 }
 
-public struct _stdlib_fd_set {
-  var _data: [UInt32]
+public struct __stdlib_fd_set<T: UnsignedInteger> {
```

Comment:
I see now why it is required -- we are passing the buffer directly to the select syscall.  Existing code would work everywhere except on 64-bit big endian platforms, right?

Instead of making it generic, which feels unnecessary to me -- and allowing creating invalid combinations of types for a platform, could we conditionally define a typealias for the underlying type (`UInt32` on 32-bit, `UInt64` on 64-bit)?  You could probably even just use a plain `UInt` without any conditionals.